model/src/oauth/current_application_info/mod.rs

```diff
@@ -12,24 +16,64 @@ pub struct CurrentApplicationInfo {
     pub cover_image: Option<String>,
     pub description: String,
     pub guild_id: Option<GuildId>,
+    /// Public flags of the application.
+    pub flags: ApplicationFlags,
     pub icon: Option<String>,
     pub id: ApplicationId,
     pub name: String,
     pub owner: User,
     pub primary_sku_id: Option<SkuId>,
+    /// URL of the application's privacy policy.
+    pub privacy_policy_url: Option<String>,
     #[serde(default)]
     pub rpc_origins: Vec<String>,
     pub slug: Option<String>,
     pub summary: String,
     pub team: Option<Team>,
+    /// URL of the application's terms of service.
+    pub terms_of_service_url: Option<String>,
     pub verify_key: String,
 }
 
 #[cfg(test)]
 mod tests {
-    use super::{CurrentApplicationInfo, GuildId, SkuId, Team, User};
+    use super::{ApplicationFlags, CurrentApplicationInfo, GuildId, SkuId, Team, User};
     use crate::{id::ApplicationId, id::UserId, oauth::id::TeamId};
+    use serde::{Deserialize, Serialize};
     use serde_test::Token;
+    use static_assertions::{assert_fields, assert_impl_all};
+    use std::{fmt::Debug, hash::Hash};
+
+    assert_fields!(
+        CurrentApplicationInfo: bot_public,
+        bot_require_code_grant,
+        cover_image,
+        description,
+        guild_id,
+        flags,
+        icon,
+        id,
+        name,
+        owner,
+        primary_sku_id,
+        privacy_policy_url,
+        rpc_origins,
+        slug,
+        summary,
+        team,
+        terms_of_service_url,
+        verify_key
+    );
+
+    assert_impl_all!(
+        CurrentApplicationInfo: Clone,
+        Debug,
+        Deserialize<'static>,
```

Comment:
You should be able to use an anonymous lifetime here (`'_`)

Reply:
I essentially looked at ones you wrote to write this, also rustc says '_ is a reserved lifetime name

Reply:
You're right, it's the struct lifetime that can be anonymous